src/containers/search_result.jsx

```diff
@@ -71,11 +71,11 @@ const VisibilityCheckbox = props => (
   />
 );
 
-
```

Comment:
Do not delete this empty line. I usually keep two empty lines after `imports`.

Reply:
Finished